TAO/TAO_IDL/fe/idl.ypp

```diff
@@ -2193,7 +2193,10 @@ literal
         }
         | IDL_WSTRING_LITERAL
         {
-          $$ = idl_global->gen ()->create_expr ($1);
+          char *wide_string = $1;
```

Comment:
I think this is fine (and I wonder how it was broken for so long, but I guess wstring is rarely used).  The only issue I can see, which isn't a major one, is that it would be nice to have a UTL_WString (or equivalent) so that the code for a WSTRING_LITERAL was similar to a STRING_LITERAL.  STRING_LITERAL uses the sval (not strval) token type.

Reply:
Yes I noticed that too. I don't know if that's worth the time right now though.